scripts/binary/binary-cleanup.js

```diff
@@ -114,13 +113,51 @@ const getDependencyPathsToKeep = async (buildAppDir) => {
   return [...Object.keys(esbuildResult.metafile.inputs), ...entryPoints]
 }
 
+const createServerEntryPointBundle = async (buildAppDir) => {
+  const unixBuildAppDir = buildAppDir.split(path.sep).join(path.posix.sep)
+  const entryPoints = [path.join(unixBuildAppDir, 'packages/server/index.js')]
+  const esbuildResult = await esbuild.build({
```

Comment:
```suggestion
  // Build the binary entry point ignoring anything that happens in the server-entry since that will be in the v8 snapshot
  const esbuildResult = await esbuild.build({
```